app/views/layouts/_flash.html.erb

```diff
@@ -1,5 +1,5 @@
 <% if flash[:error] %>
-  <div class="alert alert-danger" role="alert">
+  <div class="alert alert-banner alert-danger" role="alert">
```

Comment:
@JPrevost - At some point I'd like to have "danger" replaced with a variable that is appropriate to the type (one of: error, warning, success, info).